copy_and_nstrip_dll.ps1

```diff
@@ -31,6 +31,14 @@ else {
 $scriptDir = Split-Path -Parent -Path $MyInvocation.MyCommand.Definition
 $managedLibsFolder = $scriptDir + "\ManagedLibs"
 
+if (!(Test-Path $managedLibsFolder)) {
+    New-Item -ItemType Directory -Path $managedLibsFolder
+    Write-Output "ManagedLibs folder created successfully."

```

Comment:
I've been using `Write-Host` instead of `Write-Output`, for consistency could you replace this one to `Write-Host` please?

Reply:
sure, fixed in db363f21d482e2a79b4f01aecd8a8eb18a1b6380